ckan/tests/controllers/test_organization.py

```diff
@@ -209,6 +210,25 @@ def test_anon_user_trying_to_delete_fails(self):
                                            id=self.organization['id'])
         assert_equal(organization['state'], 'active')
 
+    @helpers.change_config('ckan.auth.create_unowned_dataset', False)
```

Comment:
Can you add a quick test for the case when `ckan.auth.create_unowned_dataset = True`? Basically check that `owner_org` is None just to make sure that that code is tested.